web3/_utils/module_testing/eth_module.py

```diff
@@ -1096,6 +1096,27 @@ async def test_async_eth_syncing(self, async_w3: "Web3") -> None:
             assert is_integer(sync_dict['currentBlock'])
             assert is_integer(sync_dict['highestBlock'])
 
+    @pytest.mark.asyncio
+    async def test_async_eth_get_storage_at(
+        self, async_w3: "Web3", emitter_contract_address: ChecksumAddress
+    ) -> None:
+        storage = await async_w3.eth.get_storage_at(emitter_contract_address, 0) # type: ignore
+        assert isinstance(storage, HexBytes)
+
+    @pytest.mark.asyncio
+    async def test_async_eth_get_storage_at_ens_name(
+        self, async_w3: "Web3", emitter_contract_address: ChecksumAddress
+    ) -> None:
+        with ens_addresses(async_w3, {'emitter.eth': emitter_contract_address}):
+            storage = await async_w3.eth.get_storage_at('emitter.eth', 0) # type: ignore
+            assert isinstance(storage, HexBytes)
+
+    @pytest.mark.asyncio
+    async def test_async_eth_get_storage_at_invalid_address(self, async_w3: "Web3") -> None:
+        coinbase = async_w3.eth.coinbase
```

Comment:
Based on the test failures, it looks like this needs to be `await`ed

Reply:
Thanks! Fixed in f64fb5d